eval_engines/ngspice/ngspice_wrapper.py

```
@@ -36,71 +49,79 @@ def __init__(self, num_process, yaml_path, path, root_dir=None):
-        with open(fpath, 'w') as f:
+        with open(fpath, "w") as f:
             f.writelines(lines)
             f.close()
         return design_folder, fpath
 
     def simulate(self, fpath):
-        info = 0 # this means no error occurred
-        command = "ngspice -b %s >/dev/null 2>&1" %fpath
+        info = 0  # this means no error occurred
+        command = "ngspice -b %s >/dev/null 2>&1" % fpath
         exit_code = os.system(command)
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)